### PR TITLE
Documentation: Add hyperlink to return to top of file #430

### LINKED ISSUE #430: Add Return to top shortcuts to our documentations
Currently all the shortcut links in our documentations are "one way trip".
We should make it further user-friendly by adding return to top link at the end of each section.

### PR DESCRIPTION
**Pull request by Zhiyuan-Amos**
Fixes #430.

### CODE REVIEW

**Comment by yamgent** [comment]
You should also add one `Return to Top` right at the end of the document. If the section happens to be very long, then the user will have to scroll back up to press `Return to Top`.

docsend

**Comment by eugenepeh** [comment]
> Also, I added the return to top at the start of each section instead of at the end

Looks good, however this only works due to the fact that most of our sections are small (fits within the screen). Also, `Return to top` also helps to serve as an indicator that user has read to the end of the section. 

Additionally, we should take into consideration that this will be used by students of CS2103 and hence, it may be better to follow the common practice (which is to put the `Return to top` at the end of each section) . It is innovative to place `Return to top` at the start of each section, instead of at the end, but it is not widely used/accepted yet.

**Comment by CanIHasReview** [comment]
# v1

@Zhiyuan-Amos submitted v1 for review.

* [v1 1/1] [Documentation: Add hyperlink to return to top of file](https://github.com/se-edu/addressbook-level4/pull/496/commits/d6fd15adf731680ca77364dfcdd52cb2939e23e5)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/5cc85fcf11b40d1660d0b6be20a3ef90bc71c1c1...d6fd15adf731680ca77364dfcdd52cb2939e23e5))
<details>
<summary>Checkout this PR version locally</summary>

```sh
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/496/1/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you wish to fetch this PR to.

</details>

**Comment by Zhiyuan-Amos** [comment]
@eugenepeh @yamgent noted, made the necessary changes!

File: docs/LearningOutcomes.adoc [hunk 1]
```diff
@@ -262,3 +288,5 @@ Here are some things you can comment on when reviewing code:
 === Exercise: Review a PR
 
 * Review a GitHub PR created by a team member.
+
+link:#addressbook-level-4---user-guide[Return to top]
```
Comment (yamgent) [inline docs/LearningOutcomes.adoc]: Wrong link? Since this is `LearningOutcome.adoc` not `UserGuide.adoc`....

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -37,8 +37,9 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 .  Refer to the link:#features[Features] section below for details of each command.
 
-== Features
+link:#addressbook-level-4---user-guide[Return to top]
```
Comment (yamgent) [inline docs/UserGuide.adoc]: ... anyway the hyperlinks doesn't seem to be working for `UserGuide.adoc` as well. Both on GitHub, and rendering to Asciidoctor locally. The page just doesn't go back to the top.

**Comment by Zhiyuan-Amos** [comment]
This is too boring :P I'll let someone else do instead.

**Pull request closed without merging**